Dockerfile.spdk

```
@@ -0,0 +1,107 @@
+# syntax = docker/dockerfile:1.4
+ARG OS_RELEASE=9
+FROM registry.access.redhat.com/ubi$OS_RELEASE/ubi:latest as prebase
+ARG OS_RELEASE
+
+# Speed up dnf
+COPY <<EOF /etc/dnf/dnf.conf
+[main]
+gpgcheck=1
+installonly_limit=3
+clean_requirements_on_remove=True
+skip_if_unavailable=False
+best=False
+fastestmirror=True
+max_parallel_downloads=10
+tsflags=nodocs
+EOF
+
+#------------------------------------------------------------------------------
+FROM prebase as base_release_8
+ARG CENTOS_PACKAGE_VERSION="8-6"
+ARG CENTOS_MIRROR="mirror.centos.org/centos"
+
+#------------------------------------------------------------------------------
+FROM prebase as base_release_9
+ARG CENTOS_PACKAGE_VERSION="9.0-20"
+ARG CENTOS_MIRROR="mirror.stream.centos.org"
+
+#------------------------------------------------------------------------------
+FROM base_release_$OS_RELEASE as base
+
+ARG CENTOS_URL="http://$CENTOS_MIRROR/$OS_RELEASE-stream/BaseOS/x86_64/os/Packages"
+ARG CENTOS_GPG_PACKAGE="$CENTOS_URL/centos-gpg-keys-$CENTOS_PACKAGE_VERSION.el$OS_RELEASE.noarch.rpm"
+ARG CENTOS_REPO_PACKAGE="$CENTOS_URL/centos-stream-repos-$CENTOS_PACKAGE_VERSION.el$OS_RELEASE.noarch.rpm"
+
+ARG CEPH_VERSION="17.2.5"
+
+
+COPY <<EOF /etc/yum.repos.d/ceph.repo 
+[Ceph]
+name=Ceph packages for \$basearch
+baseurl=http://download.ceph.com/rpm-$CEPH_VERSION/el\$releasever/\$basearch
+enabled=1
+gpgcheck=1
+type=rpm-md
+gpgkey=https://download.ceph.com/keys/release.asc
+
+[Ceph-noarch]
+name=Ceph noarch packages
+baseurl=http://download.ceph.com/rpm-$CEPH_VERSION/el\$releasever/noarch
+enabled=1
+gpgcheck=1
+type=rpm-md
+gpgkey=https://download.ceph.com/keys/release.asc
+
+[ceph-source]
+name=Ceph source packages
+baseurl=http://download.ceph.com/rpm-$CEPH_VERSION/el\$releasever/SRPMS
+enabled=1
+gpgcheck=1
+type=rpm-md
+gpgkey=https://download.ceph.com/keys/release.asc
+EOF
+
+RUN \
+    --mount=type=cache,target=/var/cache/dnf \
+    --mount=type=cache,target=/var/lib/dnf \
+    dnf remove -y dnf-plugin-subscription-manager \
+    && dnf install -y \
+        $CENTOS_GPG_PACKAGE \
+        $CENTOS_REPO_PACKAGE
+
+#------------------------------------------------------------------------------
+FROM base as build
+
+ARG SPDK_VERSION
+
+ARG PKGDEP_ARGS
+ARG CONFIGURE_ARGS
+
+ARG MAKEFLAGS
+ARG RPM_RELEASE
+
+WORKDIR /src
+COPY . .
+
+RUN \
+    --mount=type=cache,target=/var/cache/dnf \
+    --mount=type=cache,target=/var/lib/dnf \
+    --mount=type=cache,target=/root/.cache/pip \
+    dnf install -y \
+        rpm-build \
+        git \
```

The `git-core` RPM has fewer dependencies, and the main `git` RPM installs more scripts (that depend on Perl):
```suggestion
        git-core \
```

---

Good one! Thanks!